src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx

```diff
@@ -30,6 +35,12 @@ const PageTemplate: FC<PageTemplateProps> = ({ data, metadata }): JSX.Element =>
   return (
     <div className="okp4-dataverse-portal-dataverse-component-page-template-main">
       <div className="okp4-dataverse-portal-dataverse-page-template-left-side-wrapper">
+        <div className="okp4-dataverse-portal-dataverse-back">
+          <button className="okp4-dataverse-portal-dataverse-back-button" onClick={backToDataverse}>
+            <Icon name="arrow-left" />
+          </button>
+          <span className="okp4-dataverse-portal-dataverse-back-text">Dataverse</span>
```

Comment:
To keep consistency, could you add "dataverse" to common i18n? even if translation is the same for our 3 proposed languages